### ISSUE TITLE
Output on Amplify console is digital noise and not the pretty UI in your docs

### ISSUE THREAD

**Issue by stevensacks** [aws-amplify/amplify-console#327]
My output looks nothing like the screenshot on the page.

This is what it's supposed to look like (a nice UI list):
![image](https://user-images.githubusercontent.com/162252/69002972-63308280-093d-11ea-8523-784030a9117a.png)

But this is what mine looks like (noise garbage):
<img width="1070" alt="Screen Shot 2019-11-17 at 13 23 25" src="https://user-images.githubusercontent.com/162252/69002978-80fde780-093d-11ea-91de-e1f0a93b79d4.png">

What might be causing this? How can I resolve it?

**Comment by stevensacks** [comment]
@Joycehao19 Since this issue is unrelated Cypress not running at all, I decided it needed its own thread.

**Comment by Joycehao19** [comment]
Hi @stevensacks , sorry that you face this issue. Can you provide the sample code that we could test with?

**Comment by stevensacks** [comment]
Which part of the code do you want? The amplify.yml?

I can't post the entire repo. It's got our AWS codes, etc.

**Comment by stevensacks** [comment]
I just created a brand new Amplify Create React App project from scratch and got the same thing. Ugly noise in the test panel. I'm using yarn. Do you think that might have something to do with it?

**Comment by stevensacks** [comment]
Here's my package.json. Do you think it might be one of the libraries that's causing an incompatibility? Maybe testing-library, or fontawesome?

```json
{
  "name": "survey",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test --env=jsdom",
    "eject": "react-scripts eject",
    "amplify-modelgen": "node amplify/scripts/amplify-modelgen.js",
    "amplify-push": "node amplify/scripts/amplify-push.js",
    "analyze": "source-map-explorer 'build/static/js/*.js'",
    "cypress": "cypress open",
    "upd": "ncu -u yarn && yarn start",
    "test-cov": "yarn test --coverage --watch",
    "test-cov-view": "browser-sync start --server --port 5150 --files 'coverage/lcov-report/index.html' --startPath 'coverage/lcov-report'"
  },
  "husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  },
  "lint-staged": {
    "src/**/*.js": [
      "eslint --fix --max-warnings=0",
      "prettier --write",
      "git add"
    ],
    "src/**/*.json": [
      "eslint --fix",
      "prettier --write",
      "git add"
    ],
    "src/**/*.scss": [
      "sass-lint-auto-fix",
      "sass-lint -v",
      "git add"
    ]
  },
  "jest": {
    "coverageReporters": [
      "lcov"
    ],
    "collectCoverageFrom": [
      "src/**/*.js",
      "!src/index.js",
      "!src/amplify.js",
      "!src/font-awesome/*.js",
      "!src/utils/platform.js"
    ]
  },
  "dependencies": {
    "@aws-amplify/core": "^2.2.1",
    "@aws-amplify/datastore": "^1.0.3",
    "@fortawesome/fontawesome-svg-core": "^1.2.26",
    "@fortawesome/pro-regular-svg-icons": "^5.12.0",
    "@fortawesome/pro-solid-svg-icons": "^5.12.0",
    "@fortawesome/react-fontawesome": "^0.1.8",
    "@sentry/browser": "^5.11.0",
    "@types/jest": "^24.0.25",
    "@types/node": "^13.1.5",
    "@types/react": "^16.9.17",
    "@types/react-dom": "^16.9.4",
    "amplify-app": "^2.11.0",
    "aws-amplify": "^2.2.1",
    "aws-amplify-react": "^3.1.2",
    "core-js": "^3.6.2",
    "date-fns": "^2.9.0",
    "date-fns-timezone": "^0.1.4",
    "deep-object-diff": "^1.1.0",
    "element-scroll-polyfill": "^1.0.1",
    "local-storage": "^2.0.0",
    "lodash": "^4.17.15",
    "node-sass": "^4.13.0",
    "platform": "^1.3.5",
    "prop-types": "^15.7.2",
    "react": "^16.12.0",
    "react-dom": "^16.12.0",
    "react-scripts": "3.3.0",
    "regenerator-runtime": "^0.13.3",
    "source-map-explorer": "^2.2.1",
    "spark-md5": "^3.0.0",
    "typescript": "^3.7.4",
    "uuid": "^3.3.3",
    "validator": "^12.1.0"
  },
  "devDependencies": {
    "@testing-library/cypress": "^5.0.2",
    "@testing-library/jest-dom": "^4.2.4",
    "@testing-library/react": "^9.3.2",
    "@testing-library/react-hooks": "^3.2.1",
    "@testing-library/user-event": "^7.1.2",
    "@typescript-eslint/eslint-plugin": "^2.15.0",
    "@typescript-eslint/parser": "^2.15.0",
    "cypress": "^3.8.1",
    "eslint-config-prettier": "^6.9.0",
    "eslint-plugin-cypress": "^2.8.1",
    "eslint-plugin-jest": "^23.3.0",
    "eslint-plugin-jest-dom": "^1.4.2",
    "eslint-plugin-prettier": "^3.1.2",
    "eslint-plugin-react-hooks": "^2.3.0",
    "husky": "^4.0.3",
    "ini": "^1.3.5",
    "inquirer": "^7.0.3",
    "jest-localstorage-mock": "^2.4.0",
    "jest-mock-console": "^1.0.0",
    "lint-staged": "^9.5.0",
    "prettier": "^1.19.1",
    "react-test-renderer": "^16.12.0",
    "sass-lint": "^1.13.1",
    "sass-lint-auto-fix": "^0.21.0"
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}
```

**Comment by adriencg** [comment]
@stevensacks Since you can repro w/ a toy app, mind creating a public github repo with it and including your amplify.yml (the build spec) so we can try it out locally?

I just created a new react app and my test output is the pretty version you expected but aren't seeing.
